core/src/main/java/com/klaviyo/coresdk/networking/requests/IdentifyRequest.kt

```diff
@@ -28,10 +28,11 @@ internal class IdentifyRequest (
         }
      */
     override fun buildKlaviyoJsonQuery(): String {
+        addAnonymousIdToProps(properties)
```

Comment:
For example you don't want this called on the main thread.

Reply:
Also is this code thread safe? Does it need to be synchronized?

Reply:
True true. With the current setup this is only called on the batcher thread, so its already off the main thread. For the current setup this is totally safe, but if we expand the threading solution in the future then it would be good to have this synchronized